tasks/main.yml

```diff
@@ -1,48 +1,45 @@
-- debug:
-    msg: "Install environment {{ ansible_distribution }} {{ ansible_distribution_version }}"
-
-#
-# Automation Setup
-#
-# Parameters are:
-#   - automation_replicated_setting
-#   - automation_replicated_conf
-#   - automation_extra_files
-#
+###
+### Automation Setup
+###
+### Parameters are:
+###   - replicated_automation_settings_file
+###   - replicated_automation_conf_file
+###   - replicated_automation_extra_files
+###
 
 - name: Check if settings file should be copied
-  local_action: stat path='{{automation_replicated_setting}}'
+  local_action: stat path='{{replicated_automation_settings_file}}'
   register: settings
-  when: automation_replicated_setting is defined
+  when: replicated_automation_settings_file is defined
 
 - name: Copy settings file
   copy:
     dest: "/etc/settings.conf"
```

Comment:
How about a less generic name here and maybe place it in the tmp folder.  Also this file should be copied using `template`, right?